hack/tests/e2e-ansible-molecule.sh

```diff
@@ -33,12 +33,22 @@ else
 fi
 KUSTOMIZE_PATH=${KUSTOMIZE} TEST_OPERATOR_NAMESPACE=default molecule test -s kind
 
-cd $TMPDIR
-KUSTOMIZE_PATH=${KUSTOMIZE}
+rm -rf $KUSTOMIZE
+cd $TMPDIR/
+rm -rf memcached-molecule-operator
```

Comment:
You’re removing the test project here, unless I’m reading this wrong. What is being tested?

Reply:
The e2e ansible molecule tests check 2 scenarios/mock. The first scenario is the `memcached-molecule-operator` with the kind option. Then, the second test whcih is what we are fixing here is the mock in `test/ansible` which is a set of the mocks to ensure advanced features such as the project works with collections created by the users, plugins and etc.

See that in the following line we are pushing the /test/ansible mock and using it. The removal here is because we have not so longer kustomized installed globally then it will be installed for  `memcached-molecule-operator` and we need get it re-installed/in a clean state for the next mock data scenario.